vm/chef/cookbooks/orocrm/files/orocrm

```diff
@@ -75,7 +75,13 @@ php bin/console oro:install -vv \
   --env=prod \
   --no-interaction \
 
-# Assign permission to Apache user
+echo "Configure Supervisor..."
+envsubst '\$ORO_PROJECT' < /opt/c2d/supervisor.conf \
+  >> /etc/supervisor/supervisord.conf
+
+echo "Restarting Supervisor..."
```

Comment:
Please, move supervisor restart to line 95 (after restarting apache) otherwise job will be running with no OroCRM properly set up.

Reply:
Can you please also add a `systemctl stop apache2` before at the line 47 (before installing orocrm). The way script works now until Oro is set up, user see an apache empty index page.